test/npb/ft/npadmana/DistributedFFT.chpl

```diff
@@ -340,7 +340,7 @@ prototype module DistributedFFT {
       Iterate over the range ``r`` but in an offset manner based
       on the locale id.
   */
-  iter offset(r: range) { halt("Serial offset not implemented"); }
+  iter offset(r: range): nothing { halt("Serial offset not implemented"); }
```

Comment:
Again, these iterators should yield ints, for consistency with their parallel counterparts.